.github/ISSUE_TEMPLATE/update-zookeeper.md

```diff
@@ -0,0 +1,68 @@
+---
+name: Update ZooKeeper
+about: >-
+  This template contains instructions specific to updating this product and/or
+  container image(s).
+title: >-
+  chore(zookeeper): Update container images ahead of Stackable Release XX.(X)X
+labels: []
+projects: ['stackabletech/10']
+assignees: ''
+---
+
+Part of #xxx.
+
+> [!TIP]
+> Please add the `scheduled-for/20XX-XX` label.
+
+```[tasklist]
+### Update tasks
+- [ ] Update `versions.py` to reflect the agreed upon versions in the spreadsheet (including the removal of old versions).
+- [ ] Upload new version (see `zookeeper/upload_new_zookeeper_version.sh`).
+- [ ] Update `versions.py` to the latest supported version of JVM (base and devel).
+- [ ] Update other dependencies if applicable (eg: jmx_exporter, etc).
+- [ ] Check other operators (getting_started / kuttl) for usage of the versions. Add the PR(s) to the list below.
+- [ ] Update the version in demos. Add the PR(s) to the list below.
+```
+
+```[tasklist]
+### Related Pull Requests
+- [ ] _Link to the docker-images PR (product update)_
+- [ ] _Link to [hdfs-utils](https://github.com/stackabletech/hdfs-utils/) PR (if applicable)_
```

Comment:
Question: does zookeeper use hdfs-utils? As far as I know it contains the group-mapper, opa-authorizer and topology provider (all relevant to hdfs)

Reply:
I think this was some copy-pasta (same as the hive one). Will fix